README.md

```diff
@@ -25,6 +25,10 @@ npx lerna bootstrap
 
 ## Set up
 
+Before starting you need to set up couple of things
+1- Create a mongo db instance somewhere and store the information in your env variables following the template.env
+2- Set up scrum-app-local.com domain in your /etc/hosts pointing to localhost. Mandatory for facebook oauth
```

Comment:
maybe we need to create an image for this?  to avoid changes in local env ... or just do not test it in local env

Reply:
Ill look into it